docs/FRSKY_R9.md

```diff
@@ -252,12 +252,12 @@ Note: Python3 is a prerequisite to use the ELRS UARTUpload.py script.
 
 2. Download or git clone the [ExpressLRS](https://github.com/ExpressLRS/ExpressLRS) repository. If you have subversion installed: Since you only need the src/python folder, you could use the svn github interface to get just what you need. From the command line type something like this: "svn export https://github.com/ExpressLRS/ExpressLRS/tree/master/src/python"
 
-3. Connect a USB-TTL serial adapter to your computer and install a driver if it is not recognized.  
+3. Connect a USB-TTL serial adapter to your computer and install a driver if it is not recognized. Some CP210X based adapters have been found to not work, if you have one and are encountering errors please use a different USB-TTL adapter.  
```

Comment:
Can you incorporate this sentence into the note below?  Perhaps something like:

'Note: If you are having trouble flashing and are using a CP210X adapter, please use a different USB-TTL adapter.  If you continue to have problems or don't have a USB-TTL serial adapter, you should be able to use your flight controller instead - see step 8.'